src/components/ScoreBoard/Header/Header.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Header = ({ title, totalPlayers }) => <header>
+  <h1>{ title }</h1>
+  <span className="stats">
+    Players: {totalPlayers}
+  </span>
+</header>;
```

Comment:
@arvin-viar, sorry I've missed this one. We should also wrap the JSX here in parenthesis.